tests/system/CMakeLists.txt

```diff
@@ -335,6 +351,15 @@ ENDIF()
                     1 # DMC
                     TRUE)
 
+  QMC_RUN_AND_CHECK(short-bccH_1x1x1_ae-dmc-all_sdj
+                    "${CMAKE_SOURCE_DIR}/tests/solids/bccH_1x1x1_ae"
+                    qmc_short_vmc_dmc_all
+                    qmc_short_vmc_dmc_all.in.xml
+                    1 16
+                    BCC_H_DMC_SCALARS
+                    1 # DMC
+                    TRUE)
+
   IF(NOT QMC_CUDA)
   LIST(APPEND BCC_H_RMC_SCALARS "totenergy" "-1.8420 0.0011")
```

Comment:
Move the two QMC_RUN_AND_CHECK after line 363     IF(NOT QMC_CUDA) .
GPU has no all-electron move support.